copier/cli.py

```diff
@@ -189,8 +190,16 @@ def data_switch(self, values: StrSeq) -> None:
                 Each value in the list is of the following form: `NAME=VALUE`.
         """
         for arg in values:
-            key, value = arg.split("=", 1)
-            self.data[key] = value
+            if "=" in arg:
+                key, value = arg.split("=", 1)
+                self.data[key] = value
+            else:
+                if not Path(arg).exists():
```

Comment:
Should we use [`Path(arg).isfile()`](https://docs.python.org/3/library/os.path.html#os.path.isfile) instead to be more precise?

Reply:
I agree

Reply:
LGTM